prog/vm/update_ipv6.rb

```diff
@@ -22,7 +22,7 @@ def vm_host
     )
 
     write_params_json
-    vm_host.sshable.cmd("sudo host/bin/setup-vm reassign-ip6 #{vm.inhost_name}", stdin: JSON.generate({storage: vm.storage_secrets}))
+    vm_host.sshable.cmd("sudo host/bin/setup-vm reassign-ip6 #{vm.inhost_name} #{Prog::Vm::Nexus.setup_vm_options_str(vm.strand.stack.first)}", stdin: JSON.generate({storage: vm.storage_secrets}))
```

Comment:
Maybe while pushing things into a helpers in a third space (such as VmHost, Vm, or some combination), this will de-awkardify this strange method class dependency between two progs.